README.md

```diff
@@ -61,52 +61,52 @@ Download and install Titanium SDKs
 
 Installs a specific version of the Titanium SDK. If no version is specified, it assumes the latest.
 
-    ti sdk install
+    titanium sdkinstall
```

Comment:
carefull: you remove some spaces in all those commands

Reply:
it's too late for the time i wake up this morning. it's fixed and my last action for today. hopefully 😄